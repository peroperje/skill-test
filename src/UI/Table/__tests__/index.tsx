import React from 'react';
import { create } from 'react-test-renderer';

import { Table, TableRow, TableBody, TableCell, TableHead } from '../index';

describe('Table', () => {
  describe('Table', () => {
    it('Snapshot', () => {
      const component = create(
        <Table>
          <></>
        </Table>
      );
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
  describe('TableRow', () => {
    it('Snapshot', () => {
      const component = create(
        <TableRow>
          <></>
        </TableRow>
      );
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
  describe('TableBody', () => {
    it('Snapshot', () => {
      const component = create(
        <TableBody>
          <></>
        </TableBody>
      );
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
  describe('TableCell', () => {
    it('Snapshot', () => {
      const component = create(
        <TableCell>
          <></>
        </TableCell>
      );
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
  describe('TableHead', () => {
    it('Snapshot', () => {
      const component = create(
        <TableHead>
          <></>
        </TableHead>
      );
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
  describe('Integration', () => {
    it('Snapshot', () => {
      const component = create(
        <Table>
          <TableHead>
            <TableCell>Some header</TableCell>
          </TableHead>
          <TableBody>
            <TableCell>Something</TableCell>
          </TableBody>
        </Table>
      );
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
