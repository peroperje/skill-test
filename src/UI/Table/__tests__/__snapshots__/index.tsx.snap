// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Integration Snapshot 1`] = `
<table
  className="MuiTable-root"
>
  <thead
    className="MuiTableHead-root"
  >
    <th
      aria-sort={null}
      className="MuiTableCell-root MuiTableCell-head"
      scope="col"
    >
      Some header
    </th>
  </thead>
  <tbody
    className="MuiTableBody-root"
  >
    <td
      aria-sort={null}
      className="MuiTableCell-root MuiTableCell-body"
    >
      Something
    </td>
  </tbody>
</table>
`;

exports[`Table Table Snapshot 1`] = `
<table
  className="MuiTable-root"
/>
`;

exports[`Table TableBody Snapshot 1`] = `
<tbody
  className="MuiTableBody-root"
/>
`;

exports[`Table TableCell Snapshot 1`] = `
<td
  aria-sort={null}
  className="MuiTableCell-root"
/>
`;

exports[`Table TableHead Snapshot 1`] = `
<thead
  className="MuiTableHead-root"
/>
`;

exports[`Table TableRow Snapshot 1`] = `
<tr
  className="MuiTableRow-root"
/>
`;
